patches/Terraria/Configuration.targets.patch

```diff
@@ -6,13 +6,14 @@
      <TargetFramework>net40</TargetFramework>
 -    <Configurations>Debug;Release;ServerDebug;ServerRelease</Configurations>
 -    <AssemblySearchPaths>$(AssemblySearchPaths);{GAC}</AssemblySearchPaths>
+-    <PlatformTarget>x86</PlatformTarget>
 +    <Configurations>WindowsDebug;WindowsRelease;WindowsServerDebug;WindowsServerRelease;LinuxRelease;LinuxServerRelease;MacRelease;MacServerRelease</Configurations>
-     <PlatformTarget>x86</PlatformTarget>
++    <PlatformTarget>x64</PlatformTarget>
      <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
      <Optimize>true</Optimize>
 +  </PropertyGroup>
 +  <PropertyGroup Condition="$(Configuration.Contains('Windows'))">
-+    <DefineConstants>$(DefineConstants);WINDOWS;XNA</DefineConstants>
++    <DefineConstants>$(DefineConstants);WINDOWS;FNA</DefineConstants>
```

Comment:
Why are you forcing 64-bit and FNA in legacy config?

Reply:
Mistake from merging with Rartrin.